docs/docs/0.7.1/documentation/dps-and-data-model.md

```diff
@@ -34,10 +31,7 @@ When we start a new data-plane in Kuma, **two things** have to happen:
 1. The data-plane needs to advertise what service it is responsible for. This is what the `Dataplane` entity does.
 2. The data-plane process needs to start accepting incoming and outgoing requests.
 
-These steps are being executed in **two separate** commands:
-
-1. We register the `Dataplane` object via the `kumactl` or HTTP API.
-2. Once we have registered the DP, we can start it by running `kuma-dp run`.
+To do this, we have to create a file with a `Dataplane` definition and pass it to `kuma-dp run`. This way, data-plane will be registered in the Control Plane and Envoy will start accepting requests.
```

Comment:
Is this the recommended way, or the only possible way. I assume the old kumactl apply is still valid for the DataPlane resource. Still this is the preferred way, therefore this is what we recommend here.

Reply:
Old way is still valid, but a new one is recommended. I did not want to confuse users even more about it here. I think the old flow is preferable for integrations like Kubernetes when we convert DP from something.